### ISSUE TITLE
FluentUI TooltipHost : Behaviour of the tooltip is inconsistent in mobile and tablet devices

### ISSUE THREAD

**Issue by Manjunatha123** [microsoft/fluentui#20967]
### Environment Information

- **Package version(s)**: "@fluentui/react": "^8.46.0"
- **Browser and OS versions**: Safari iOS 

### Please provide a reproduction of the bug in a codepen:
Note: Could not replicate the issue in codepen but happening in our application.

#### Actual behavior:
When a tooltip is present in a page with large content, the tooltip for a word is not working on the first touch and any number of consecutive touches on the same word. If the user clicks somewhere else on the page and clicks on the word tooltip is populating in the mobile devices.
When the page is initially loaded and user directly clicks(touches) word with tooltip, the tooltip is not shown to the user.
If the page reloads due to change in the query parameter like 
if the current url is https://<tenant>.sharepoint.com/sites/<sitename>/page-name1.aspx?param1='abc'
and it is changed to https://<tenant>.sharepoint.com/sites/<sitename>/page-name2.aspx?param1='def'
and user clicks on the tooltip word as soon as the page loads, tooltip is not shown to the user on the first click and any consecutive clicks on that word.
Please find the usage as below.
```
const tooltipStyles: Partial<ITooltipHostStyles> = { root: { display: 'inline-block' } };
const calloutProps: ICalloutProps = { gapSpace: 0 };
const tooltipProps: ITooltipProps = { delay: TooltipDelay.long };
   <TooltipHost
       id={uniqueID}
       content={tooltipContent} //html content
       styles={tooltipStyles}
       calloutProps={calloutProps}
       tooltipProps={tooltipProps} 
       className={styles.tooltipStyles}
       closeDelay={300}
       onTooltipToggle={(isTooltipVisible) => { document.body.style.setProperty('-webkit-overflow-scrolling', isTooltipVisible ? 'unset':'touch')}}>
<span className={"tooltip"}>"tooltipWord"</span>
</TooltipHost>
```

For debugging purpose we had put the onClick and onTouchStart both the events to the span element in the above code.
We observed that if first thing user touches on the page is the tooltip word, the tooltip is not shown to the user and both the onClick and on touch start event is fired. For the case when the tooltip is shown to the user only onTouchStart event is fired.

`<span className={"tooltip"} onClick={()=>console.log('onClick'}}" onTouchStart={()=>console.log('onTouchStart')}>tooltipWord"</span>`

We thought this may be issue due to focus and tried focusing all the possible divs we had in the DOM but it did not help.
Tried triggering a click event on the div nearby to get the focus in componentDidMount as well, it also did not help.

In our application we have almost 100 paragraphs in a page and  3 - 10 tooltips per paragraphs. For example consider the below paragraph
Lorem Ipsum is **simply** dummy text of the printing and **typesetting** industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and **scrambled** it to make a type specimen book. It has survived not only five centuries

Whenever user touches the word simply, typesetting, scrambled tooltips with content related to the word are shown.

#### Expected behavior:
User should be able to see the tooltip on the first touch of the word wrapped with `<TooltipHost>` in ipad and iphone devices.
This behaviour is not observed in the desktop for hover over the word with tooltip.

### Priorities and help requested:

Are you willing to submit a PR to fix? (Yes, No) No

Requested priority: (Blocking, High, Normal, Low) Normal

Products/sites affected: (if applicable)

**Comment by KshamaShrey** [comment]
We are also facing similar issues with mobile devices. When the page is initially loaded and user directly clicks(touches) word with tooltip, the tooltip is not shown to the user first time .

**Comment by Suj5432** [comment]
Seeing the same issue in mobile devices - iPhone and iPad where we use TooltipHost for displaying the tooltips present in the page. 

Observed below things:
- Once the page loads and user clicks on a tooltip, it does not display the tooltip content.
- On the same page (without refreshing the page) if user clicks on another tooltip, the tooltip content appears.

Is it due to the click and touch event triggered at the same time?

**Comment by tista-chatterjee** [comment]
Facing same issue in mobile devices for tooltips display in our application on first page load.

**Comment by Namrata99sah** [comment]
The same issue is happening in our application, when the page loads for first time, after first click the tooltip is not showing. Only after 2nd click is shows any tooltip.

**Comment by Manjunatha123** [comment]
Any updates on this issue?
production site is affected due to this issue.

**Comment by layershifter** [comment]
Can you please provide an isolated repro on Codepen/CodeSandbox with repro steps? 🙏

**Comment by Manjunatha123** [comment]
As mentioned we could not replicate the issue with codesandbox.
We are trying to get that replicated

**Comment by Manjunatha123** [comment]
We could not replicate the issue in codesandbox.io.
Since on initial page load onClick event is triggered for the tooltip target, We created ref and triggered `TooltipRef.current.show` on click of span element.
But there is one more issue when large content is present, 
Steps
1. Goto the page with tooltip https://codesandbox.io/s/youthful--e9p25-forked-huj9b in iPhone device.
2. Open tooltip CodeSandbox, tooltip opens
3. Scroll outside tooltip.
4. Open one more CodeSandbox tooltip.
5. try to scroll within the tooltip user is not able to scroll within the tooltip.

Repro is here [https://codesandbox.io/s/youthful--e9p25-forked-huj9b](https://codesandbox.io/s/youthful--e9p25-forked-huj9b)

Could we repurpose this ticket with the new issue or a new ticket needs to be created?
@layershifter @msft-fluent-ui-bot

**Comment by layershifter** [comment]
^ @spmonahan can you please check this one?
